src/blockchain/node/wallet/subchain/statemachine/Scan.cpp

```diff
@@ -271,31 +251,26 @@ auto Scan::Imp::work() noexcept -> bool
         opentxs::print(current()))
         .Flush();
 
+    auto makeWork = [this](Vector<ScanStatus>&& vec) {
```

Comment:
It is now used twice, dependent on the state of the object and it is no longer even nameless - you could as well have defined this as a private function :-) This won't stop me fro approving, though.

Reply:
done;)